README.rst

```diff
@@ -566,8 +567,10 @@ Scala
 Data Science
 ~~~~~~~~~~~~
 
+* `conda-forge/cookiecutter-feedstock`_: Instruction on installing cookiecutter from conda-forge.
```

Comment:
We do have a section in the install docs: https://github.com/audreyr/cookiecutter/blame/master/docs/installation.rst#L19-L30